gen2-yolo/host-decoding/models/README.md

```diff
@@ -0,0 +1,2 @@
+Download pretrained models from https://drive.google.com/file/d/1kNqH7U07mXWKib8jQ1ULg4Mz03I6EtQk/view?usp=sharing using `python download.py`
```

Comment:
Can we just run this download by default if the nn file is not found? Or at least move the download note to README.md in the parent directory, so that the script doesn't fail after following the setup instructions

Reply:
And same for other download.py files in gen2-yolo

Reply:
Rather added it to instructions than manual download. Other READMEs already had it.